project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=0.13.13
```

Comment:
`sbt.version` locks down the SBT version. If you don't have this defines, SBT launcher will take whatever is installed in your system and it is quite unreliable. I, for instance, haven't updated SBT setup for a long time and project failed to start because of this. `build.properties` fixed it.

`.idea` folder is just an intellij idea project files. You don't really want them in a git repo, so i ignored them :)